sdk/storage/storage-file-share/test/leaseclient.spec.ts

```diff
@@ -96,7 +96,8 @@ describe("LeaseClient", () => {
   });
 
   it("invalid duration for acquireLease", async () => {
```

Comment:
May worth add more cases to cover newly added functions.

Reply:
Will do in follow-up PR.